### PR TITLE
Add initial set of configuration files.

### PR DESCRIPTION
**Pull request by nanliu**

### CODE REVIEW

File: moduleroot/.travis.yml [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+- 1.6.3
+- 1.7.0
+<%- if settings.addons? -%>
+<%= pad_yaml(settings.addons) %>
+<%- end -%>
+env:
+  global:
+  - ORG_PATH=/home/travis/gopath/src/github.com/intelsdi-x
+  - SNAP_PLUGIN_SOURCE=/home/travis/gopath/src/github.com/${TRAVIS_REPO_SLUG}
+  - GO15VENDOREXPERIMENT=1
```
Comment (candysmurf) [inline moduleroot/.travis.yml]: is go1.5+ still supported?

**Comment by candysmurf** [comment]
@mjbrender @sarahjhh: Do you like to review some templates and CONTRIBUTING.md file?
@intelsdi-x/snap-maintainers: your inputs are welcome!

**Comment by candysmurf** [comment]
LGTM

File: moduleroot/.travis.yml (hunk 1 above)
Comment (nanliu) [inline moduleroot/.travis.yml]: good catch, I can drop this since we no longer have go 1.5.4 in the matrix

File: moduleroot/CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+If you have questions feel free to contact the [maintainers](https://github.com/intelsdi-x/snap/blob/master/README.md#maintainers) by tagging them: @intelsdi-x/plugin-maintainers.
+
+## Contributing Examples
+The most immediately helpful way you can benefit this project is by cloning the repository, adding some further examples and submitting a pull request.
+
+Have you written a blog post about how you use [snap](http://github.com/intelsdi-x/snap) and/or this plugin? Send it to us [on slack](https://intelsdi-x.slack.com/)!
+
+## Contribute Elsewhere
+This repository is one of **many** plugins in **snap**, a powerful telemetry framework. See the full project at http://github.com/intelsdi-x/snap
```
Comment (sarahjhh) [inline moduleroot/CONTRIBUTING.md]: capitalize Snap

File: moduleroot/CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+* Open a pull request for the appropriate project.
+* Contributors will review your pull request, suggest changes, and merge it when it’s ready and/or offer feedback.
+* To report a bug or issue, please open a new issue against this repository.
+
+If you have questions feel free to contact the [maintainers](https://github.com/intelsdi-x/snap/blob/master/README.md#maintainers) by tagging them: @intelsdi-x/plugin-maintainers.
+
+## Contributing Examples
+The most immediately helpful way you can benefit this project is by cloning the repository, adding some further examples and submitting a pull request.
+
+Have you written a blog post about how you use [snap](http://github.com/intelsdi-x/snap) and/or this plugin? Send it to us [on slack](https://intelsdi-x.slack.com/)!
```
Comment (sarahjhh) [inline moduleroot/CONTRIBUTING.md]: capitalize Snap here too

File: moduleroot/CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+4. [Thank You](#thank-you)
+
+This repository has dedicated developers from Intel working on updates. The most helpful way to contribute is by reporting your experience through issues. Issues may not be updated while we review internally, but they're still incredibly appreciated.
+
+## Contributing Code
+**_IMPORTANT_**: We encourage contributions to the project from the community. We ask that you keep the following guidelines in mind when planning your contribution.
+
+* Whether your contribution is for a bug fix or a feature request, **create an [Issue](https://github.com/intelsdi-x/<%= repo_name %>/issues)** and let us know what you are thinking.
+* **For bugs**, if you have already found a fix, feel free to submit a Pull Request referencing the Issue you created.
+* **For feature requests**, we want to improve upon the library incrementally which means small changes at a time. In order ensure your PR can be reviewed in a timely manner, please keep PRs small, e.g. <10 files and <500 lines changed. If you think this is unrealistic, then mention that within the issue and we can discuss it.
```
Comment (sarahjhh) [inline moduleroot/CONTRIBUTING.md]: insert 'to' : In order **to** ensure your PR...

**Comment by sarahjhh** [comment]
LGTM 👍

**Comment by nanliu** [comment]
@jcooklin, I added the list of managed files in pluginsync.yml per your request. Let me know if you can think of anything else we want to include for this first pass:

See below for example:
https://github.com/intelsdi-x/snap-plugin-publisher-file/pull/16/files#diff-d5f4e999e69b6da52648680eb4138eb6

**Comment by jcooklin** [comment]
LGTM.

As a separate PR it would be helpful to have more examples that include how to add exceptions.  For instance how would we define a one-off travis file for a given plugin.

**Pull request merged**